Control/public/environment/environmentsPage.js

```diff
@@ -61,10 +68,72 @@ export const content = (model) => h('.scroll-y.absolute-fill.text-center', [
  * @param {Array.<Environment>} list
  * @return {vnode}
  */
-const showContent = (model, list) => (list && Object.keys(list).length > 0)
-  ? h('.scroll-auto', environmentsTable(model, list))
-  : h('h3.m4', ['No environments found.']);
+const showContent = (model, list) =>
+  (list && Object.keys(list).length > 0)
+    ? h('.scroll-auto', environmentsTable(model, list))
+    : h('h3.m4', ['No environments found.']);
+
+/**
+ * Show list of create env request to AliECS core as a table
+ * @param {Object} model
+ * @param {array} requests List of requested stored in the backend
+ * @param {Date} now Current date according to backend
+ */
+const showRequests = (model, requests, now) =>
+  (requests && requests.length > 0)
+    ? requestsTable(model, requests, now)
+    : h('h3.m4', ['No requests found.']);
 
+/**
+ * Calculate duration from two dates and pretify result
+ * @param {Date} now Current date
+ * @param {Date} startDate Date when a request was cached
+ */
+const durationDisplay = (now, startDate) => {
```

Comment:
While the idea of having a timer is good, I think we should not compare only on request refresh. 
We should have a frontend timer which refreshes itself after 1 second. Reason behind is that currently on my own setup I have a 2 minutes refresh period which means for me it simply displays 0s for 2 minutes

Reply:
Okay, then in such case I'd just display created timestamp, and leave the timer as future improvement.

Reply:
I agree